app/js/components/orderEntry/OrdersTable.jsx

```diff
@@ -1,30 +1,62 @@
-import React from 'react';
+import React, { PureComponent } from 'react';
 import PropTypes from 'prop-types';
+import { connect } from 'react-redux';
+import format from 'date-fns/format';
 import Accordion from './Accordion';
-import allOrdersMock from './allOrdersMock.json';
 import OrderHeader from './OrderHeader';
 import LabOrderDetails from './LabOrderDetails';
 import DrugOrderDetails from './DrugOrderDetails';
+import fetchOrder from '../../actions/fetchOrder';
 
-const OrdersTable = () => (
-  <React.Fragment>
-    {allOrdersMock.orders.map(order => (
-      <Accordion
-        title={<OrderHeader status={order.status} orderable={order.orderable} />}
-        key={order.id}>
-        {order.type === 'drugorder' ? (
-          <DrugOrderDetails
-            dosingInstructions={order.dosingInstructions}
-            dispense={order.dispense}
-            activeDates={order.activeDates}
-            orderer={order.orderer}
-          />
-        ) : (
-          <LabOrderDetails urgency={order.urgency} orderer={order.orderer} />
-        )}
-      </Accordion>
-    ))}
-  </React.Fragment>
-);
+export class OrdersTable extends PureComponent {
+  componentDidMount() {
+    this.props.dispatch(fetchOrder(null, 5, this.props.patient.uuid));
+  }
+  componentDidUpdate(prevProps, prevState, snapshot) {
+    if (this.props.patient.uuid !== prevProps.patient.uuid) {
```

Comment:
you do not seem to use the `prevState` and `snapshot` variables here, please take it out.